### PR TITLE
Blog app - controllers specs

### PR DESCRIPTION
**Pull request by uno36**
### Project requirements
- Add RSpec as a gem to your project.
- Create a Request spec file for all your controllers.
- Make sure that for each action you check:
    - If response status was correct.
    - If a correct template was rendered.
    - If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/users_spec.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      get '/users'
+      expect(response).to render_template('users/index')
+      expect(response).to render_template(layout: 'layouts/application')
+    end
+
+    it 'assigns @users with all users' do
+      get '/users'
+      expect(assigns(:users)).to eq(User.all)
+    end
+  end
```
Comment (anagudelogu) [inline spec/requests/users_spec.rb]: - [ ] According to the requirements:

image

Please include a test for the placeholder text `response.body` 🤓 🔧

File: spec/requests/users_spec.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      get "/users/#{@user.id}"
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the show template' do
+      get "/users/#{@user.id}"
+      expect(response).to render_template('users/show')
+      expect(response).to render_template(layout: 'layouts/application')
+    end
+  end
```
Comment (anagudelogu) [inline spec/requests/users_spec.rb]: - [ ] The same with this #show action, please include a test for the placeholder text in `response.body`. 🤓 🔧

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @uno36, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — APPROVED** [review]
Hi @uno36 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* All tests passed 👍 
* Well written PR 👍 
* No linter errors 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**